tests/test_menu.py

```diff
@@ -1,21 +1,8 @@
-from djangocms_alias.utils import is_versioning_enabled
-
 from .base import BaseAliasPluginTestCase
 
 
 class AliasMenuTestCase(BaseAliasPluginTestCase):
 
-    def test_alias_pages_have_no_menu_nodes(self):
```

Comment:
I have added this back in, but with the first assert removed, as I think this is no longer relevant now that we are using the admin Category changelist view, since this doesnt include the Aliase menu. But let me know what you think this is incorrect